src/diff/index.js

```diff
@@ -381,9 +381,9 @@ function diffElementNodes(
  * @param {any} value
  * @param {import('../internal').VNode} vnode
  */
-export function applyRef(ref, value, vnode) {
+export function applyRef(ref, value, vnode, oldRef) {
```

Comment:
On my phone so maybe I’m a bit lost here but what’s this argument for?

Reply:
oops, me being dumb good spot 👍